core/jobs/types/base_validation_errors.py

```diff
@@ -35,24 +35,15 @@
 (base_models,) = models.Registry.import_models([models.Names.BASE_MODEL])
 
 
-class BaseAuditError(job_run_result.JobRunResult):
-    """Base class for model audit errors."""
+class BaseValidationError(job_run_result.JobRunResult):
+    """Base class for model validation errors."""
 
-    def __init__(
-        self,
-        message: str,
-        model_or_kind: Union[base_models.BaseModel, str],
-        model_id: Optional[str] = None
-    ) -> None:
+    def __init__(self, message: str, model: base_models.BaseModel) -> None:
```

Comment:
Perhaps better to always pass both model_kind and model_id?

Reply:
Replied above.